kakmerge

```diff
@@ -15,9 +15,12 @@ readonly KAKMERGE_CMDS='
         Search for the contents of the main selection in all clients (local, base, remote, merged)
     } %{ eval -draft %{
         exec -save-regs "" <space> *
-        exec -client local n
-        exec -client base n
-        exec -client remote n
+        try %{ exec -client local n }
+        try %{ exec -client base n }
+        try %{ exec -client remote n }
+        eval -client local echo ""
+        eval -client base echo ""
+        eval -client remote echo ""
```

Comment:
echoing empty string is a way to get rid of (2) on the screenshot, if you have better ideas how to achieve this, let me know :)

Reply:
No other way to do that, it's a topic that's been floating in the community for a while but was never addressed.

What I don't like about this approach is that it will hide _any_ message, even those we might not necessarily want to hide (e.g. errors). I guess this will have to do for now.